seed/challenges/intermediate-bonfires.json

```diff
@@ -639,7 +639,8 @@
       "tests": [
         "expect(drop([1, 2, 3, 4], function(n) {return n >= 3; })).to.eqls([3, 4]);",
         "expect(drop([1, 2, 3], function(n) {return n > 0; })).to.eqls([1, 2, 3]);",
-        "expect(drop([1, 2, 3, 4], function(n) {return n > 5; })).to.eqls([]);"
+        "expect(drop([1, 2, 3, 4], function(n) {return n > 5; })).to.eqls([]);",
+        "expect(drop([1, 2, 3, 7, 4], function(n) { return > 5; })).to.eqls([7, 4]);"
```

Comment:
I'd prefer the space as it promotes readability and fits inline with our code style. But, it does break the pattern that is already there. I think this should not hold off merging this PR.